### PR TITLE
Login with Azure & user model

### PR DESCRIPTION
**Pull request by slkendell-mt**

### CODE REVIEW

File: backoffice/src/Models/FormModels/BaseEntityForm.cs
Comment (SamBrownMT) [inline backoffice/src/Models/FormModels/BaseEntityForm.cs:1]: Is this now redundant?

File: backoffice/src/Models/ViewModels/UserView.cs
Comment (SamBrownMT) [inline backoffice/src/Models/ViewModels/UserView.cs:1]: Understandably we can't use a base model for the user model and user form model, but can we add it here or would this cause too much inconsistency across the codebase?

Comment (slkendell-mt) [reply backoffice/src/Models/ViewModels/UserView.cs:1]: protected BaseEntityView(BaseEntity entity)
        {
            Created = entity.Created;
            LastModified = entity.LastModified;
            LastModifiedBy = entity.LastModifiedByUser?.Name ?? "Unknown";
        }

Technically we could, but we'd have to have an additional constructor to take not a BaseEntity. 

To make User Work as a baseEntity we would need to add some conditional Include if not me logic (Which isn't really worth it)

File: backoffice/src/Models/FormModels/BaseEntityForm.cs
Comment (slkendell-mt) [reply backoffice/src/Models/FormModels/BaseEntityForm.cs:1]: It was, but I've moved the Id logic here. Worth having this for later if we ever decide to add more common fields

**Review by SamBrownMT — APPROVED** [review]

**Review by SamBrownMT — APPROVED** [review]

**Pull request merged**